guide/src/test/scala/org/zalando/grafter/QuickStart.scala

```diff
@@ -0,0 +1,81 @@
+package org.zalando.grafter
+
+object QuickStart extends UserGuidePage { def is = "Quick Start".title ^ s2"""
+
+### Installation
+
+To use Grafter you need add it as a dependency in your sbt build settings:
+
+```scala
+libraryDependencies += "org.zalando" %% "grafter" % "$version"
+
+// you also need the macros plugin for the grafter annotations
+addCompilerPlugin("org.scalamacros" % "paradise" % "2.1.0" cross CrossVersion.full)
```

Comment:
`2.1.1` or perhaps a more recent version?

Reply:
The macros site says `2.1.0`: http://docs.scala-lang.org/overviews/macros/paradise.html

Reply:
Then they are outdated: http://search.maven.org/#artifactdetails%7Corg.scalamacros%7Cparadise_2.12.4%7C2.1.1%7Cjar

Reply:
Thanks I updated the version.